crates/bevy_ui/src/widget/text.rs

```diff
@@ -432,8 +433,8 @@ pub fn text_system(
             continue;
         };
         let scale_factor = match scale_factors_buffer.entry(camera_entity) {
-            Entry::Occupied(entry) => *entry.get(),
-            Entry::Vacant(entry) => *entry.insert(
+            bevy_utils::Entry::Occupied(entry) => *entry.get(),
```

Comment:
I don't understand why these diffs exist.

Reply:
This import was only used when these features are enabled, so rather than having the messy cfg for the tiny `utils` import, I just inlined it.